gatsby-config.js

```diff
@@ -630,6 +630,59 @@ module.exports = {
                                 title: "Release notes",
                                 path: "documentation/adobe-journey-optimizer/release-notes"
                             }
+                            {
+                                title: "AJO Powered In-app messaging",
```

Comment:
Since we're making it a sub-topic within AJO, we can just make this "In-app messaging" instead?